install/kubernetes/helm/istio/values.yaml

```diff
@@ -295,6 +295,10 @@ global:
     # If using stackdriver tracer outside GCP, set env GOOGLE_APPLICATION_CREDENTIALS to the GCP credential file.
     tracer: "zipkin"
 
+    # Specify whether use TCP loopback address as Envoy admin endpoint.
+    # If it's set to false, unix domain socket will be used for admin endpoint.
```

Comment:
I think this is the wrong logic. This shouldn't be an either/or setting. Envoy should be configured to use UDS unconditionally for the admin interface. The option should be about enabling a TCP proxy to the admin interface in pilot-agent.